src/Jenkins/Build.php

```diff
@@ -65,13 +65,20 @@ public function getInputParameters()
     {
         $parameters = array();
 
-        if (!property_exists($this->build->actions[0], 'parameters')) {
-            return $parameters;
-        }
-
-        foreach ($this->build->actions[0]->parameters as $parameter) {
-            $parameters[$parameter->name] = $parameter->value;
-        }
+		// Parameters may not always be the first definition. Search for 'hudson.model.ParametersAction' to find
+		// real parameters.
+		foreach ($this->build->actions as $action) {
+			if (isset($action->_class) && $action->_class === 'hudson.model.ParametersAction') {
```

Comment:
I know we don't have tools to automatically check code style, but can you replace your tabs by 4 spaces as we follow psr2/3 (cf https://github.com/jenkins-khan/jenkins-php-api#coding-standards)